dev-requirements.in

```diff
@@ -1,7 +1,9 @@
-pip-tools==4.5.*
+pip-tools==5.1.*
 pytest==5.3.*
 pytest-flake8==1.0.*
 pytest-isort==0.3.*
 pytest-mypy==0.5.*
 pytest-env==0.6.*
 pytest-cov==2.8.*
+mock==4.0.*
+google-cloud-storage==1.28.*
```

Comment:
Why is this a development time dependency? This should be *only* in requirements.ini. If it is missing from your dev env, there is a different problem but packages that are alraedy specified in requirements.in should not be here.